### ISSUE TITLE
Filter by User Group

### ISSUE THREAD

**Issue by dragonmantank** [phpmentoring/testapp#21]
Be able to filter people by the User Groups they attend

**Comment by matthewtrask** [comment]
@dragonmantank is this still a thing?

**Comment by dragonmantank** [comment]
Yes, I think so. We've also got #95 (Languages). Timezone (already implemented), and #20 / #55 (Location) that are good pivot points for searching.

**Comment by matthewtrask** [comment]
Yes it is as in its still open, or there is enough points to search by?

**Comment by dragonmantank** [comment]
Yes in that it's a useful metric to search by :)

It doesn't currently have any code to support it though. The users will need a new form element to add in their user group, the API updated to be able to search by it, and the UI updated to allow for the searching.

**Comment by matthewtrask** [comment]
ok then ill take care of it.
